src/cdk/observers/observe-content.spec.ts

```diff
@@ -120,6 +120,57 @@ describe('Observe content', () => {
   });
 });
 
+describe('Observe content injectable', () => {
```

Comment:
Since all the logic is in the injectable, maybe we should put all of the tests under this suite?

Reply:
I wanted to leave the above tests alone to show that this doesn't change the behavior of the directive. Yes, we could add more tests here as well, but it felt like it would just be retesting the same logic